### ISSUE TITLE
About the CE loss

### ISSUE THREAD

**Issue by vhientran** [hanjiale/HCRP#3]
Hi @hanjiale,

Thank you very much for releasing the source code of your great work.
I see your source code is very clear and easy to follow. I only wonder about calculating the CE loss at this [line](https://github.com/hanjiale/HCRP/blob/183c62c3e704a029fa55f6f666db85f4b7ddef34/models/hcrp.py#L80).
If possible, Could you please explain why we need to get the min value of logits and then get the max value on N+1 values? As I think we can only simply get the max value on N values...
I am looking forward to hearing from you. Many thanks!

**Comment by hanjiale** [comment]
Thank you for your attention to our work! We follow the implementation in FewRel https://github.com/thunlp/FewRel/blob/master/models/proto.py#L47

**Comment by vhientran** [comment]
I got it. Thank you for your reply!
